bin/critterget.py

```diff
@@ -134,10 +138,10 @@ def getAppSummary():
 
     AppDict = {}
     for appId in gdata.keys():
-        printstring = u'{} MessageType="AppSummary" appId={} '.format(myruntime, appId)
+        printstring = "%s MessageType=\"AppSummary\" appId=%s " % (myruntime, appId)
```

Comment:
Why are we reverting back to old string formatting?  These have unicode problems.